.github/actions/test-dependencies/action.yml

```diff
@@ -29,3 +29,8 @@ runs:
       uses: ./.github/actions/monero
       with:
         version: ${{ inputs.monero-version }}
+
+    - name: Run a Monero Wallet-RPC Client
+      uses: ./.github/actions/monero-wallet-rpc
+      with:
```

Comment:
Remove this with. We can just always use monero-wallet-rpc 0.18.2. The reason we test older daemons is to test our successful handling of old TX types. That isn't relevant here.